src/components/Synthetics/MarketCard/MarketCard.tsx

```diff
@@ -5,9 +5,11 @@ import Tooltip from "components/Tooltip/Tooltip";
 import {
   MarketInfo,
   getAvailableUsdLiquidityForPosition,
+  getCurrentOpenInterestUsd,
   getMarketIndexName,
   getMarketPoolName,
-  getMaxOpenInterest,
+  getMaxAllowedReservedUsd,
```

Comment:
small thing, can we use `getMaxReservedUsd` and `getOpenInterestUsd` to keep it more consistent with contracts code. this is not necessary, but similar names often help to find UI logic that corresponds to contracts logic

Reply:
Updated!